specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```diff
@@ -8843,7 +8843,55 @@
         "code": {
           "type": "string",
           "readOnly": true,
-          "description": "One of a server-defined set of error codes."
+          "description": "One of a server-defined set of error codes.",
```

Comment:
Since searchindex.json needs these as well, I recommend having a common json file you ref from both.